lib/coarse_op_mma.cuh

```diff
@@ -0,0 +1,633 @@
+#pragma once
```

Comment:
Since this file doesn't contain any kernels, and doesn't have any actual CUDA, perhaps a more appropriate name would be `coarse_op_mma_launch.hpp` or `coarse_op_mma_launch.h`?